tactics/class_tactics.ml

```diff
@@ -181,36 +171,6 @@ let unify_resolve ~with_evars flags h diff = match diff with
   Clenv.res_pf ~with_evars ~with_classes:false ~flags clenv
   end
 
-(** Application of a lemma using [refine] instead of the old [w_unify] *)
-let unify_resolve_refine flags h diff =
```

Comment:
Wasn't this necessary also for unifall?

Reply:
I don't know, and I don't know the current state of unifall. In the worst case this can be reintroduced, it's just that it's dead code now.